doc/changes/changes_1.3.0.md

```diff
@@ -1,9 +1,13 @@
-# Cloud Storage Extension 1.3.0, released 2021-??-??
+# Cloud Storage Extension 1.3.0, released 2021-07-27
 
-Code name: 
+Code name: Bug Fixes and Refactorings
 
 ## Summary
 
+In this releases we added support for exporting Hive compatible Parquet schema. Also, this release fixes several bugs. We fixed bug with delimited identifiers and missing decimal trailing zeros when doing export.
```

Comment:
> this release fixes several bugs. We fixed bug
Sounds a bit strange to be that there is fixed bug twice. Maybe you can rephrase?

Reply:
Updated